src/sagemaker_training/environment.py

```diff
@@ -1176,6 +1197,15 @@ def is_modelparallel_enabled(self):  # type: () -> bool
         """
         return self._is_modelparallel_enabled
 
+    @property
+    def is_smddp_smddpmprun_installed(self):  # type: () -> bool
```

Comment:
If we remove importing the library, just rename this method to `is_smddpmprun_installed`

Reply:
Yes, updated it.